lib/Dialect/RTL/RTLOps.cpp

```diff
@@ -459,6 +459,66 @@ static LogicalResult verifyInstanceOp(InstanceOp op) {
       return failure();
   }
 
+  // If the referenced moudle is internal, check that input and result types are
```

Comment:
nit: the verify function is a bit long for my taste, so I would break out the new code into another function.

Reply:
just personal preference so feel free to ignore.

Reply:
I have split the function